plugins/spz/io/Util.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <map>
+
+#include <pdal/util/Utils.hpp>
+#include <load-spz.h>
+
+namespace spz {
+
+CoordinateSystem getCoordinateSystem(const std::string &name)
+{
+    std::map<std::string, spz::CoordinateSystem> names = {
```

Comment:
bubble these names up to the ProgramArgs so the user can know what the possibilities are.